cloudflare_worker_version SIGSEGV on import

### Confirmation

- [x] This is a bug with an existing resource and is not a feature request or enhancement. Feature requests should be submitted with Cloudflare Support or your account team.
- [x] I have searched the issue tracker and my issue isn't already found.
- [x] I have replicated my issue using the latest version of the provider and it is still present.

### Terraform and Cloudflare provider version

Terraform v1.7.5 on darwin_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v5.9.0

### Affected resource(s)

- cloudflare_worker_version

### Terraform configuration files

```hcl
resource "cloudflare_worker_version" "workers" {
  account_id         = ""
  worker_id          = ""

  modules = [
    {
      name         = "index.js"
      content_file = each.value.script_path
      content_type = "application/javascript+module"
    }
  ]

  main_module = "index.js"
}
```

### Link to debug output

N/A

### Panic output

╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵


Stack trace from the terraform-provider-cloudflare_v5.9.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x23a0ebb]

goroutine 60 [running]:
github.com/cloudflare/terraform-provider-cloudflare/internal/services/worker_version.(*WorkerVersionResource).Read(0xc00013e080, {0x4e3e998, 0xc00055ecf0}, {{{{0x4ec4730, 0xc0002b7740}, {0x3b61900, 0xc0002b6f00}}, {0x4efff10, 0xc0003a3c70}}, 0x0, ...}, ...)
	github.com/cloudflare/terraform-provider-cloudflare/internal/services/worker_version/resource.go:172 +0x8db
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadResource(0xc0000ff408, {0x4e3e998, 0xc00055ecf0}, 0xc00049a5a0, 0xc00063f5d0)
	github.com/hashicorp/terraform-plugin-framework@v1.15.0/internal/fwserver/server_readresource.go:156 +0xbae
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ReadResource(0xc0000ff408, {0x4e3e998?, 0xc00055ec00?}, 0xc000025ae0)
	github.com/hashicorp/terraform-plugin-framework@v1.15.0/internal/proto6server/server_readresource.go:71 +0x592
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0xc000296aa0, {0x4e3e998?, 0xc00054d920?}, 0xc000457c70)
	github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov6/tf6server/server.go:860 +0x2f6
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x4782340, 0xc000296aa0}, {0x4e3e998, 0xc00054d920}, 0xc000479b00, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:651 +0x1a9
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000228200, {0x4e3e998, 0xc00054d890}, 0xc00040b5c0, 0xc0003aa9f0, 0x8746258, 0x0)
	google.golang.org/grpc@v1.72.1/server.go:1405 +0x103b
google.golang.org/grpc.(*Server).handleStream(0xc000228200, {0x4e3f350, 0xc0003b21a0}, 0xc00040b5c0)
	google.golang.org/grpc@v1.72.1/server.go:1815 +0xbaa
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.72.1/server.go:1035 +0x7f
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 17
	google.golang.org/grpc@v1.72.1/server.go:1046 +0x125

Error: The terraform-provider-cloudflare_v5.9.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.


### Expected output

Import the item.

### Actual output

╷
│ Error: Request cancelled
│
│ The plugin6.(*GRPCProvider).ReadResource request was cancelled.
╵

### Steps to reproduce

1. tf init
2. tf import

### Additional factoids

_No response_

### References

_No response_

--- a/internal/services/worker_version/resource.go
+++ b/internal/services/worker_version/resource.go
@@ -169,26 +169,30 @@ func (r *WorkerVersionResource) Read(ctx context.Context, req resource.ReadReque
 	data.Assets = assets
 
 	// Refresh content_sha256 on each module
-	moduleNameMap := make(map[string]*WorkerVersionModulesModel, len(*stateModules))
-	for _, mod := range *stateModules {
-		moduleNameMap[mod.Name.ValueString()] = mod
+	moduleNameMap := make(map[string]*WorkerVersionModulesModel)
+	if stateModules != nil {
+		for _, mod := range *stateModules {
+			moduleNameMap[mod.Name.ValueString()] = mod
+		}
 	}
-	for _, mod := range *data.Modules {
-		contentBase64 := mod.ContentBase64.ValueString()
-		content, err := base64.StdEncoding.DecodeString(contentBase64)
-		if err != nil {
-			resp.Diagnostics.AddError("Refresh Error", err.Error())
-			return
-		}
-		contentSHA256, err := calculateStringHash(string(content))
-		if err != nil {
-			resp.Diagnostics.AddError("Refresh Error", err.Error())
-			return
-		}
+	if data.Modules != nil {
+		for _, mod := range *data.Modules {
+			contentBase64 := mod.ContentBase64.ValueString()
+			content, err := base64.StdEncoding.DecodeString(contentBase64)
+			if err != nil {
+				resp.Diagnostics.AddError("Refresh Error", err.Error())
+				return
+			}
+			contentSHA256, err := calculateStringHash(string(content))
+			if err != nil {
+				resp.Diagnostics.AddError("Refresh Error", err.Error())
+				return
+			}
 
-		mod.ContentSHA256 = types.StringValue(contentSHA256)
-		if stateMod, ok := moduleNameMap[mod.Name.ValueString()]; ok {
-			mod.ContentFile = stateMod.ContentFile
+			mod.ContentSHA256 = types.StringValue(contentSHA256)
+			if stateMod, ok := moduleNameMap[mod.Name.ValueString()]; ok {
+				mod.ContentFile = stateMod.ContentFile
+			}
 		}
 	}
 

